ArxTypeTraits/type_traits.h

```diff
@@ -378,15 +389,12 @@ namespace std {
     using result_of = details::result_of<Sig>;
 
 
-    template <class T>
-    void swap(T& a, T& b)
-    {
-        T t = move(a);
-        a = move(b);
-        b = move(t);
-    }
-
+} // namespace std
 
+#if __has_include(<initializer_list>)
+#include <initializer_list>
+#else
+// ArxContainer might also define initializer_list, so don't redefine
 #ifndef ARX_TYPE_TRAITS_INITIALIZER_LIST_DEFINED
 #define ARX_TYPE_TRAITS_INITIALIZER_LIST_DEFINED
 
```

Comment:
Please add `std` namespace to `initializer_list`

Reply:
Good point.